### ISSUE TITLE
Главная страница. Название добавленной станции метро в графе «Метро» отображается не целиком 

### ISSUE THREAD

**Issue by ArtemyBabitsky** [ArtemyBabitsky/Test-afisha.mail.ru-#1]
_Как воспроизвести:_
1.Открыть главную страницу сайта: https://afisha.mail.ru/spb/cinema/kinoafisha;
2.В поле «Метро» ввести название станции петербургского метро;
3.Выбрать найденную станцию метро.

_Фактический результат:_
Название станции метро в поле "Метро" отображается не целиком (некорректно).
![1](https://cloud.githubusercontent.com/assets/23294267/20038642/3f04655a-a448-11e6-8af5-dfb0b1a1efbe.jpg)

_Ожидаемый результат:_
Отображение названия добавленной станции производится корректно, пользователь видит название станции метро, которую выбрал.

**Comment by marinapenkova** [comment]
Да, действительно баг. Только я переформулировала бы название ошибки, без использования "не целиком", "некорректно"
